pynest/nest/lib/hl_api_helper.py

```diff
@@ -472,23 +472,33 @@ def show_help_with_pager(hlpobj, pager):
 
     # reading ~/.nestrc lookink for pager to use.
     if pager is None:
-        # open ~/.nestrc
-        rc = open(os.environ['HOME'] + '/.nestrc', 'r')
-        for line in rc:
-            rctst = re.match(r'^\s?%', line)
-            if rctst is None:
-                pypagers = re.findall(
-                    r'\s?/page\s?<<\s?/command\s?\((.*)\).*', line)
-                if pypagers:
-                    pager = pypagers[0]
-                    break
-                else:
-                    pager = 'less'
-        rc.close()
-
+        # check if .netsrc exist
+        rc_file = os.environ['HOME'] + '/.nestrc'
```

Comment:
You should use `os.path.join(os.environ['HOME'], '.nestrc')` for portable path construction; applies in other places as well.